vaa-core/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "private": true,
+  "name": "vaa-core",
+  "version": "1.0.0",
+  "scripts": {
+    "build": "yarn --modules-folder ./node_modules tsc --build && yarn tsc-esm-fix",
+    "lint": "eslint . ",
+    "lint:fix": "prettier --write . && eslint --fix . ",
+    "format": "prettier --write .",
+    "test": "vitest run",
+    "test:watch": "vitest"
+  },
+  "type": "module",
+  "module": "./build/index.js",
+  "types": "./build/index.d.ts",
+  "exports": {
+    "import": "./build/index.js"
+  },
+  "devDependencies": {
```

Comment:
The `vaa` modules now all have the same copy-pasted dev deps but in a subsequent update, it'd be best to harmonize these using a shared config for deps as well as linter, prettier and TS configs. See [Issue 624](https://github.com/OpenVAA/voting-advice-application/issues/624#issuecomment-2451440981).